### ISSUE TITLE
[BUG] rename `_make_hierachical` to `_make_hierarchical`

### ISSUE THREAD

**Issue by fkiraly** [alan-turing-institute/sktime#2195]
Not a bug strictly speaking, but at typo.

`_make_hierachical` should be renamed to `_make_hierarchical`, since it's called "hierarchical" time series data, not "hierachical" time series data.

Should happen before 0.11.0 where it can happen without deprecation.

**Comment by aiwalter** [comment]
changes in private methods and functions can imho always happen without deprecation as long as the funtionality is not changed?

**Comment by fkiraly** [comment]
true, you are right. Would never need deprecation since it's a private method.
Still we should probably do it quickly since we don't have many refs to this.

**Comment by Vasudeva-bit** [comment]
I would like to work on this, can you explain a little bit more!
